platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -63,8 +75,8 @@ class AboutModal extends Component {
   renderTableRow(item) {
     return (
       <tr key={item.name}>
-        <td>{item.name}</td>
-        <td>
+        <td data-cy={item.testLabel}>{item.name}</td>
+        <td data-cy={item.testId}>
```

Comment:
For most of these tests, we only super care that the body of the modal contains some expected text. We can create a single `data-cy="about-modal-body"` on the Modal's body, and then assert that it contains our expected text. This has a few benefits:

- We don't need to update our row render logic to include extra values that are only meant for testing
- The entire modal body layout can change, but the tests will still pass
    - IE. we don't care about the layout, we only care about the content